lib/rules/no-magic-numbers.js

```diff
@@ -75,6 +194,22 @@ module.exports = {
 						type: "boolean",
 						default: false,
 					},
+					ignoreEnums: {
+						type: "boolean",
+						default: false,
+					},
+					ignoreNumericLiteralTypes: {
+						type: "boolean",
+						default: false,
+					},
+					ignoreReadonlyClassProperties: {
+						type: "boolean",
+						default: false,
+					},
+					ignoreTypeIndexes: {
+						type: "boolean",
+						default: false,
+					},
```

Comment:
Can you also update types for this rule?

https://github.com/eslint/eslint/blob/865dbfed6cbade8a22756965be256da317801937/lib/types/rules.d.ts#L2821-L2842

Reply:
Done.

Reply:
I don't see changes in `eslint/lib/types/rules.d.ts` in this PR, maybe you forgot to push the commit?

Reply:
Maybe so. Added in https://github.com/eslint/eslint/pull/19561/commits/d0b9b64bb8587ece2134308b63e1544cba2c1bee